src/status_im/contexts/profile/edit/name/utils.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im.contexts.profile.edit.name.utils
+  (:require [clojure.string :as string]
+            [utils.i18n :as i18n]))
+
+(def min-length 5)
+
+(def max-length 24)
+
+(def emoji-regex
+  (new
+   js/RegExp
+   #"(\u00a9|\u00ae|[\u2000-\u3300]|\ud83c[\ud000-\udfff]|\ud83d[\ud000-\udfff]|\ud83e[\ud000-\udfff])"
```

Comment:
@smohamedjavid we have this elsewhere in the codebase?
We have an emoji picker code in the codebase, perhaps we should move this regex and other similar utilities to some namespace encompassing all things emoji related?